### PR TITLE
Add notification_request attribute to MessageCreator

### PR DESCRIPTION
**Pull request by raphaelsattler**

### CODE REVIEW

File: app/services/message_sender.rb [hunk 1]
```diff
@@ -24,12 +24,14 @@ def call
   private
 
   def create_message!
+    target_data = { target: target, target_type: target_type }
```
Comment (kaiomagalhaes) [inline app/services/message_sender.rb:27]: please rename target to targe_name. Like this: 

```
target = { name: target, type: target_type }
```

File: app/services/message_creator.rb [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
-  def initialize(provider, action, target_type, message, target, uniq)
+  def initialize(action, message, notification_request, provider, target_data, uniq)
     super()
     @action = action
-    @target_type = target_type
-    @text = message
-    @target = target
+    @notification_request = notification_request
     @provider = provider
+    @target = target_data[:target]
```
Comment (kaiomagalhaes) [inline app/services/message_creator.rb:9]: please rename `@target` to `@target_name`

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: app/services/message_creator.rb [hunk 3]
```diff
@@ -1,25 +1,27 @@
 # frozen_string_literal: true
 
 class MessageCreator < ApplicationService
-  def initialize(provider, action, target_type, message, target, uniq)
+  def initialize(action, message, notification_request, provider, target_data, uniq)
```
Comment (kaiomagalhaes) [inline app/services/message_creator.rb:4]: please rename it from `target_data` to `target`

File: app/services/message_sender.rb [hunk 4]
```diff
@@ -24,12 +24,14 @@ def call
   private
 
   def create_message!
+    target_data = { target_name: target, target_type: target_type }
```
Comment (kaiomagalhaes) [inline app/services/message_sender.rb:27]: please rename `target_data` to `target` 
please rename `target_name` to `name` 
please rename `target_type` to `type`

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: app/services/message_sender.rb (hunk 4 above)
Comment (raphaelsattler) [reply app/services/message_sender.rb:27]: @kaiomagalhaes , we already have a variable called target here, we can't rename the target_data variable without rename the target variable.

Comment (kaiomagalhaes) [reply app/services/message_sender.rb]: @raphaelsattler rename the current @target to @target_name

**Pull request merged**